src/pytorch_lightning/_graveyard/utilities.py

```diff
@@ -0,0 +1,33 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+from typing import Any
+
+
+def _patch_sys_modules() -> None:
+    # TODO: Remove in v2.0.0
+    self = sys.modules[__name__]
```

Comment:
This is only done if the module does not exist, but in this case it does.

You just need to monkeypatch the attribute to the existing one

I'll help

Reply:
Done! Thank you for the guidance!